pinot-core/src/main/java/com/linkedin/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -148,10 +153,11 @@ public DataTable processQuery(final QueryRequest queryRequest) {
       instanceResponse = new DataTable();
       instanceResponse.addException(QueryException.getException(QueryException.QUERY_EXECUTION_ERROR, e));
       TraceContext.logException("ServerQueryExecutorV1Impl", "Exception occurs in processQuery");
-      long end = System.currentTimeMillis();
-      LOGGER.info("Searching Instance for Request Id - {}, browse took: {}", requestId, requestId, (end - start));
+      queryProcessingTimer.stopAndRecord();
+
+      LOGGER.info("Searching Instance for Request Id - {}, browse took: {}", requestId, requestId, queryProcessingTimer.getDuration());
```

Comment:
that's existing code not new statement. 

That said, the log statement and incorrect so I'll fix it